### PR TITLE
Intake controls

### PR DESCRIPTION
**Pull request by tbriden**

### CODE REVIEW

File: src/main/java/frc/robot/competition/CompetitionRobotContainer.java [hunk 1]
```diff
@@ -106,6 +108,7 @@ class CompetitionRobotContainer {
 
     SmartDashboard.putData("AutoMode", autoChooser);
 
+    configureIntake();
```
Comment (tsieben) [inline src/main/java/frc/robot/competition/CompetitionRobotContainer.java:111]: I don't necessarily think we need to create a separate method to just instantiate the intake subsystem.

File: src/main/java/frc/robot/subsystems/Intake.java
Comment (tsieben) [inline src/main/java/frc/robot/subsystems/Intake.java:1]: May want to add the factory classes here instead of creating the separate commands.

**Review by tsieben — APPROVED** [review]
Left few comments on the improvements that Tim W mentioned but approved.

File: src/main/java/frc/robot/commands/IntakeNote.java
Comment (ArchdukeTim) [inline src/main/java/frc/robot/commands/IntakeNote.java:1]: Convert this class to two command factory methods in the `Intake` subsystem

**Review by ArchdukeTim — CHANGES_REQUESTED** [review]

**Review by ArchdukeTim — APPROVED** [review]

**Pull request merged**